tests/src/Unit/CodeGenerationTestCase.php

```diff
@@ -441,15 +441,13 @@ protected function assertClassSubclassOf($class, $baseClass, $message = '')
      * This will cause the class to be available for subsequent code.
      *
      * @param ClassGenerator $generator The object from with to generate the class.
-     * @param string $namespace The namespace to use for the class.
      */
-    protected function generateClass(ClassGenerator $generator, $namespace = null)
+    protected function generateClass(ClassGenerator $generator)
     {
-        $source = $generator->getClass()->getSource();
-        if (!empty($namespace)) {
-            $source = 'namespace ' . $namespace . ';' . PHP_EOL . $source;
-        }
+        $source = $generator->getClass()->generate();
```

Comment:
Sorry, may be some misunderstanding here? ```$namespace``` removed because it is not necessary here any more.